lib/assets/styles/classes.scss

```diff
@@ -459,6 +459,50 @@ a,
   overflow: hidden;
 }
 
+.card-divider {
+  background-color: var(--color-button-bg);
+  border: none;
+  color: var(--color-button-bg);
+  height: 1px;
+  margin: var(--gap-sm) 0;
+}
+
+// TABLE
+.table-container {
+  display: grid;
+  grid-template-rows: repeat(auto-fill, auto);
+  width: 100%;
+  border-radius: var(--radius-md);
+  overflow: hidden;
+}
+.table-row {
+  display: grid;
+  grid-template-columns: 1fr 4fr 1.5fr;
+}
+.table-head {
+  .table-cell {
+    background-color: var(--color-accent-contrast);
+  }
+}
+.table-cell {
+  padding: 1rem;
+  height: 100%;
+  align-items: center;
+  vertical-align: center;
+  display: flex;
+}
+.table-text {
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  display: flex;
+  span {
+    display: inline-block;
+    text-overflow: ellipsis;
+    overflow: hidden;
+  }
+}
+
```

Comment:
For this I would have a parent class for the whole table called `table` or something.

I would also incorporate the fix you have for the button's `span`s here as well since it'll probably be the case for a lot of uses of the table